frontend/src/entry-points/review.tsx

```diff
@@ -11,6 +11,12 @@ import '@unocss/reset/tailwind.css'
 import { isAppInfo } from '../utils'
 import type { ReviewResponse } from '../bindings/ReviewResponse'
 import type { ReviewRequest } from '../bindings/ReviewRequest'
+import eruda from 'eruda'
+
+if (import.meta.env.DEV) {
+  console.log('Starting eruda')
+  eruda.init()
+}
```

Comment:
if tree shaking works to remove the code inside of the if statement at build time, then do a dynamic import in inside of there: 

```
const test = import('eruda').then(e=>e.init())
```
though I don't know if dynamic imports support npm modules with vite or just relative paths to modules?